<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="LAPTOP-DULAB47M" name="TestCases.AutomationPageTest" tests="11" failures="0" timestamp="21 Nov 2020 14:08:51 GMT" time="93.031" errors="1">
  <testcase name="switchtoAlertfunctionality" time="0.000" classname="TestCases.AutomationPageTest">
    <skipped/>
  </testcase> <!-- switchtoAlertfunctionality -->
  <testcase name="geturlinnewtab" time="0.000" classname="TestCases.AutomationPageTest">
    <skipped/>
  </testcase> <!-- geturlinnewtab -->
  <testcase name="geturlinnewtab" time="12.625" classname="TestCases.AutomationPageTest"/>
  <testcase name="performallctionsfunctionality" time="14.824" classname="TestCases.AutomationPageTest"/>
  <testcase name="initialize" time="0.012" classname="TestCases.AutomationPageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Resources.base.InitializeDriver(base.java:29)
	at TestCases.AutomationPageTest.initialize(AutomationPageTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:599)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- initialize -->
  <testcase name="switchtoAlertfunctionality" time="40.912" classname="TestCases.AutomationPageTest"/>
  <testcase name="performallctionsfunctionality" time="0.000" classname="TestCases.AutomationPageTest">
    <skipped/>
  </testcase> <!-- performallctionsfunctionality -->
  <testcase name="verifyRadiobuttonfunctionality" time="0.000" classname="TestCases.AutomationPageTest">
    <skipped/>
  </testcase> <!-- verifyRadiobuttonfunctionality -->
  <testcase name="verifySugessiondropdwonfunctionality" time="0.000" classname="TestCases.AutomationPageTest">
    <skipped/>
  </testcase> <!-- verifySugessiondropdwonfunctionality -->
  <testcase name="verifyRadiobuttonfunctionality" time="14.902" classname="TestCases.AutomationPageTest"/>
  <testcase name="verifySugessiondropdwonfunctionality" time="9.756" classname="TestCases.AutomationPageTest"/>
</testsuite> <!-- TestCases.AutomationPageTest -->
